src/main/kotlin/no/nav/familie/ef/søknad/api/dto/Adresse.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.familie.ef.søknad.api.dto
+
+data class Adresse(val adresse: String,
+                   val adressetillegg: String,
```

Comment:
Hva er dette feltet brukt til?

Reply:
Vet ikke helst sikkert, men man innførte for noen år siden en ekstra adresselinje i Norge for tin som v/ og c/o. Regner med at det er det dette felte brukes til.